internal/service/common/clients/k8s/k8s.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/openshift/assisted-service/api/v1beta1"
```

Comment:
perhaps an alias to clearly identify agentv1beta1

Reply:
fixed